.vscode/launch.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
Revert this file.

Reply:
reverted, by using git checkout develop -- filename